### DISCUSSION TITLE
Actually Free @Mythical from the `display: none;` curse!

### DISCUSSION THREAD

**Discussion by FerretPaws** [stoatchat/discussions#201] [General]
He's still stuck in a white void! It seems like his settings didn't get reset at all!

**Comment by sussycatgirl** [comment]
Lol. Press ctrl+shift+i, and check the styles section - You should be able to temporarily disable the individual rules there.

![image](https://user-images.githubusercontent.com/26145882/139424588-17f00d34-3ad2-4d41-96e8-60d11402d221.png)

**Reply by FerretPaws** [reply]
I'm stuck on a chromebook that has element inspection disabled by the enterprise! I'm not able to do that ;w;

**Reply by sussycatgirl** [reply]
Oh god. No idea what to do then

**Reply by FerretPaws** [reply]
Yeahhhhh. I kinda need my profile fully wiped of settings

**Reply by sussycatgirl** [reply]
Insert said he wiped your settings already on the other thread (@insertish stop locking threads please kthxbay), so I suspect that the code which is responsible for downloading your synced settings just didn't run for some reason - Can you try opening revolt in a private tab or clearing browser data?

**Reply by FerretPaws** [reply]
Yee. I'll try that real quick

**Reply by FerretPaws** [reply]
Ayeeee, it worked! =3

**Reply by sussycatgirl** [reply]
Alright, nice!

Don't do that again lol
